mdanalysissandbox/tests/test_ross_verploegh.py

```diff
@@ -0,0 +1,10 @@
+"""Unit tests for Verploegh's new function."""
+
+from mdanalysissandbox import ross_verploegh
+
+
+def test_dot_product():
+    """Test for the dot product."""
+    vector_1 = [0, 1, 1]
+    vector_2 = [1, 1, 0]
+    assert (ross_verploegh.dot_product(vector_1, vector_2) == 1.0)
```

Comment:
equality comparison for floats is dangerous.

For the sandbox this is fine, for production code you'd use
```python
from numpy.testing import assert_almost_equal
```
and use 
```python
assert_almost_equal(ross_verploegh.dot_product(vector_1, vector_2), 1.0)
```